src/Generating/Trees.cpp

```diff
@@ -159,13 +240,13 @@ void GetTreeImageByBiome(int a_BlockX, int a_BlockY, int a_BlockZ, cNoise & a_No
 		case biColdBeach:
 		{
 			// Apple or birch trees:
-			if (a_Noise.IntNoise3DInt(a_BlockX, a_BlockY + 16 * a_Seq, a_BlockZ + 16 * a_Seq) < 0x5fffffff)
+			if (a_Noise.IntNoise3DInt(a_BlockPos.addedY(16 * a_Seq).addedZ(16 * a_Seq)) < 0x5fffffff)
```

Comment:
This looks weird. Personally, I'd pull the `a_Seq`-based offset into a local variable `seqOffset` and use a regular vector addition in all the places. But since this (`addedX().addedY().addedZ()`) is used everywhere else here, I don't mind so much.